web-ui/src/main/resources/catalog/locales/en-admin.json

```diff
@@ -632,7 +632,7 @@
     "system/clickablehyperlinks/enable-help": "If set, GeoNetwork will display clickable hyperlinks in the metadata.",
     "system/documentation": "Documentation configuration",
     "system/documentation/url": "Base manual url",
-    "system/documentation/url-help": "Base application manual url. Defaults to the official manual page (https://docs.geonetwork-opensource.org/{version}/{lang}) and can be customised to use a self hosted documentation with a custom branding. The url can contain \\{\\{lang\\}\\} placeholder, to display the manual in different languages when available, and \\{\\{version\\}\\} placeholder to use the application version.",
+    "system/documentation/url-help": "Base application manual url. Defaults to the official manual page (https://docs.geonetwork-opensource.org/{version}/{lang}) and can be customised to use a self hosted documentation with a custom branding. The url can contain \\{\\{lang\\}\\} placeholder, to display the manual in different languages when available, \\{\\{version\\}\\} placeholder to use the application version, and \\{\\{section\\}\\} placeholder to parse sub section url from manual.json of the current page.",
```

Comment:
```suggestion
    "system/documentation/url-help": "Base application manual url. Defaults to the official manual page (https://docs.geonetwork-opensource.org/{version}/{lang}) and can be customised to use a self hosted documentation with a custom branding. The url can contain \\{\\{lang\\}\\} placeholder, to display the manual in different languages when available, \\{\\{version\\}\\} placeholder to use the application version, and \\{\\{section\\}\\} placeholder to parse sub section url from manual.json of the current page. When the \\{\\{section\\}\\} placeholder is not provided, the sub section value is automatically appended to the end of the url.",
```

Reply:
@josegar74 

I have introduced a new variable name isoLang which should do the job. It wont affect the original design of that path variable of {{lang}}. We have earlier made similar change to this section by keeping two language variables

![image](https://github.com/user-attachments/assets/8e748100-7402-4c74-8dcb-6d4e18dd5ff6)


It will do the trick of lang={{isoLang}}

@ianwallen 
No worry about missing sub section variable. I have put that consideration into the coding. It handles the default well by appending section at the end. So it will not break other origanization's catalogue setup